docs/getting_started.md

```diff
@@ -203,6 +203,167 @@ See these resources for more information on clustering:
  - [Quantifying clustering behavior in Orchestrating Single Cell Analysis](https://bioconductor.org/books/release/OSCA.advanced/clustering-redux.html#quantifying-clustering-behavior)
 
 
+### Cell type annotation
+
+Processed objects may contain cell type annotations and associated metadata from one or more of the following sources.
+
+- Submitter-provided annotations
+- Automated annotations from [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html), a reference-based method ([Looney _et al._ 2019](https://doi.org/10.1038/s41590-018-0276-y)).
+- Automated annotations from [`CellAssign`](https://github.com/Irrationone/cellassign), a marker-gene based method ([Zhang _et al._ 2019](https://doi.org/10.1038/s41592-019-0529-1)).
+
+Below we provide instructions on how to access each type of cell type annotation, if present.
+
+
+#### Submitter-provided annotations
+
+To access submitter-provided annotations in the `SingleCellExperiment`, use the following command:
```

Comment:
I think we should add a note that not all libraries will contain submitter annotations. Although everything should include CellAssign/SingleR, it's much more likely that these will be missing.